server/src/chatbot.py

```diff
@@ -18,7 +18,8 @@ def chatbot(user_message):
     preprocessed_message = preprocess_input(user_message)
     rf_model, tfidf_vect = model()
     intent = predict_intent(rf_model, tfidf_vect,preprocessed_message)
-    output = match_intent_to_response(intent)
+    itent_dictionary = read_data()
```

Comment:
I didn't mean encapsulate it in a function. I mean I don't want it to read from the excel file every time a request is made. We can build the dictionary once and then search it (O(1) operation) every time a request is made.